src/joint.dia.element.js

```diff
@@ -775,7 +776,6 @@ joint.dia.ElementView = joint.dia.CellView.extend({
 
             // let the linkview deal with this event
             this._linkView.pointerup(evt, x, y);
-
             delete this._linkView;
 
```

Comment:
If this is the approach you want to take, then it makes more sense to add it back in (since they are now named), but in my implementation, batch start is called once on LinkView models' pointerdown (via its protoype) and same for ElementView.